helpdesk-message-proxy-service/src/main/java/org/symphonyoss/symphony/bots/helpdesk/messageproxy/service/TicketService.java

```diff
@@ -74,8 +75,9 @@ public Ticket createTicket(String ticketId, SymMessage message, String serviceSt
       LOGGER.error("Could not get symphony user when creating ticket: ", e);
     }
 
-    Ticket ticket = ticketClient.createTicket(ticketId, message.getStreamId(), serviceStreamId,
-        Long.valueOf(message.getTimestamp()), client);
+    Ticket ticket = ticketClient.createTicket(ticketId, message.getStreamId(), serviceStream.getId(),
+        Long.valueOf(message.getTimestamp()), client, serviceStream.getRoomDetail().getRoomAttributes().getViewHistory(),
```

Comment:
Please extract the view history flag to a variable. You should make it to timestamp as well.